tests/Functional/Net/SFTPUserStoryTest.php

```diff
@@ -543,14 +543,24 @@ public function testRmDirScratch($sftp)
 
     /**
      * @depends testRmDirScratch
+     * @group github706
```

Comment:
It does for me on Travis CI:

https://travis-ci.org/terrafrost/phpseclib/builds/69507514